certified-connectors/Polaris PSA/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2662 @@
+                    "slug": {
+                      "type": "string",
+                      "description": "slug"
+                    }
+                  },
+                  "description": "actingUser"
+                },
+                "displayText": {
+                  "type": "string",
+                  "description": "displayText"
+                },
+                "actingForUser": {
+                  "type": "string",
+                  "description": "actingForUser"
+                },
+                "authorityType": {
+                  "type": "object",
+                  "properties": {
+                    "displayText": {
+                      "type": "string",
+                      "description": "displayText"
+                    },
+                    "uri": {
+                      "type": "string",
+                      "description": "uri"
+                    }
+                  },
+                  "description": "authorityType"
+                }
+              },
+              "description": "authority"
+            },
+            "timesheetStatusUri": {
+              "type": "string",
+              "description": "timesheetStatusUri"
+            }
+          }
+        },
+        "description": "the response for webhook subscription"
+      },
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/WebhookSubscriptionResponse"
+            }
+          }
+        },
+        "summary": "WebhookSubscriptionsRestAPI",
+        "operationId": "WebhookSubscriptionsRestAPI",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "eventType": {
+                  "type": "string",
+                  "description": "eventType"
+                },
+                "targetUrl": {
+                  "type": "string",
+                  "description": "targetUrl",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                }
+              },
+              "required": [
+                "targetUrl"
+              ]
+            }
+          }
+        ],
+        "description": "Trigger to Subscribe to Web hook eventis in Polaris"
+      }
+    }
+  },
+  "definitions": {
+    "GetTimesheetSummaryResponse": {
+      "type": "object",
+      "properties": {
+        "d": {
+          "type": "object",
+          "properties": {
+            "actualPayableTimeCalculationStatus": {
+              "type": "string",
+              "description": "actualPayableTimeCalculationStatus"
```

Comment:
@repl-anand-vikram The description values of properties should be free from grammatical and spelling errors. Please update them with valid description values. 

Applies this comment for all other similar values.